src/mixins/clipboard.js

```diff
@@ -9,10 +9,21 @@ export default {
       copyToClipboard(text);
       this.alertPromptSuccess(alertMessage);
     },
-    copyURLPath(path, options) {
+    share({ title, text, url }) {
+      navigator.share({
+        title,
+        text,
+        url,
+      });
+    },
+    shareURLPath({ title, text, path }, options) {
```

Comment:
Maybe just merge into one object
```suggestion
    shareURLPath({ title, text, path, ...options }) {
```

Reply:
But this should not use options but use alertMessage? 
`      this.shareURLPath({
        title: this.userDisplayName,
        text: this.userDisplayName,
        path: `/${wallet}?referrer=${referrer}`,
        alertMessage: this.$t('tooltip_share_done'),
      });`